osi_lane.proto

```diff
@@ -854,6 +854,11 @@ message LaneBoundary
             // The current \c BoundaryPoint indicates the end of a dash.
             //
             DASH_END = 4;
+
+            // The current \c BoundaryPoint is located in the gap between
+            // two dashes on a dashed line or indicates the start or end of 
+            // a lane boundary off the dash.
+            DASH_GAP = 5;
```

Comment:
**CCB 08.12.2021 proposed documentation update:**
// The current \c BoundaryPoint is located on a gap between two dashes.
// When used to describe a first/last point of a lane boundary, it indicates that the lane boundary starts/ends in a gap.

Reply:
Done. @kmeids: The PR needs to be reviewed by someone else so that it can be merged.

Reply:
Did the approval, as it is implemented as discussed in the CCB-meeting.